### PR TITLE
NavBar + Profile Skeleton

### PR DESCRIPTION
**Pull request by nicholasrobinharrison**
### Done

- Created the NavBar component
- Linked NavBar to userContext
- Created a reusable Logo component in case we need it anywhere outside of the NavBar

### to-do

- Profile select button needs work
- Notifications button will also need to be updated eventually but is fine for now

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
 import React, { useContext } from "react";
 import { UserContext } from "../context/UserContext";
+import { NavBar } from "../components";
```
Comment (skassam21) [inline client/src/pages/Home.js]: not a fan of these relative paths, rather have it absolute from src

File: client/public/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/logo.png" />
```
Comment (skassam21) [inline client/public/index.html]: We should really have our a favicon lol (can use an online generator)

Comment (nicholasrobinharrison) [reply client/public/index.html]: updated

File: client/src/pages/Home.js (hunk 1 above)
Comment (nicholasrobinharrison) [reply client/src/pages/Home.js]: changed

File: client/.env [hunk 3]
```diff
@@ -0,0 +1 @@
+NODE_PATH=src/
```
Comment (rkerins91) [inline client/.env]: I think we should .gitignore .env files

Comment (nicholasrobinharrison) [reply client/.env]: yeah my bad i just did 'git add . '

Comment (nicholasrobinharrison) [reply client/.env]: should be gone now

File: client/src/components/Logo.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import logo from "assets/images/logo.png";
+
+export const Logo = () => {
+  const style = {
+    backgroundImage: "url(" + logo + ")",
+    backgroundSize: "contain",
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center",
+    height: "6vh",
```
Comment (ashan36) [inline client/src/components/Logo.js]: Logo is a bit too large, the image gets grainy. Is there an SVG version somewhere?

File: client/src/components/NavBar.js [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+              color="inherit"
+              fontSize="large"
+            ></IconButton>
+            <Button className={classes.profileButton}>
+              <AccountCircle />
+              <Typography className={classes.user}>
+                {user ? user.name : "profile"}
+              </Typography>
+            </Button>
+            <Menu
```
Comment (ashan36) [inline client/src/components/NavBar.js:94]: This menu doesn't display correctly for me. It renders on the left side of the profileButton and it doesn't have the little arrow yet.

File: client/src/components/Logo.js (hunk 4 above)
Comment (nicholasrobinharrison) [reply client/src/components/Logo.js]: Don't think there is an svg. Might just have to live with the grainyness

File: client/src/components/NavBar.js (hunk 5 above)
Comment (nicholasrobinharrison) [reply client/src/components/NavBar.js:94]: Just made an edit, how's it looking now?

**Pull request merged**